lib/util/DocumentExtractor.js

```diff
@@ -42,9 +42,19 @@ const extractors = ([
     methods: {
       extractFromRequest: request => {
         const documents = [];
-
-        for (let it = 0; it < request.input.body.ids.length; it++) {
-          documents.push({ _id: request.input.body.ids[it] });
+        if (request.input.body && Object.keys(request.input.body).length) {
+          for (let it = 0; it < request.input.body.ids.length; it++) {
+            documents.push({ _id: request.input.body.ids[it] });
+          }
+        } else if (request.input.args.ids) {
+          if (Array.isArray(request.input.args.ids)) {
+            documents.push({ _id: request.input.args.ids });
+            return documents;
+          }
+          const ids = request.input.args.ids.split(',');
```

Comment:
Throws a TypeError if there is an `ids` parameters in args that is neither an array nor a string.

Reply:
so what is the expected behavior if a wrong parameter is in the request?

Reply:
it should return the correct api error